types/Media.php

```diff
@@ -111,4 +111,24 @@ public function valueEditor($name, $rawvalue) {
         $html .= "</button>";
         return $html;
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function renderTagCloudLink($value, \Doku_Renderer $R, $mode, $page, $filter, $weight) {
+        $media = $this->displayValue($value);
+        if ($mode == 'xhtml' && $this->getConfig()['mime'] == 'image/') {
+            $url = wl($page, $filter);
+            $image = ml($media, ['h' => $weight, 'w' => $weight]);
+            $R->doc .= "<div style=\"height:{$weight}px; width:{$weight}px\">";
+            $R->doc .= "<a href='$url' class='struct_image' style='background-image:url(\"$image\")' title='$media'>";
+            $R->doc .= "<span class='hide_text'>$media</span>";
```

Comment:
this span isn't really needed, the ``alt`` attribute should be enough. ``$media`` should be esacaped (``hsc``)